app/renderer/components/bookmarks/bookmarksToolbar.js

```diff
@@ -196,23 +195,23 @@ class BookmarksToolbar extends React.Component {
   mergeProps (state, ownProps) {
     const currentWindow = state.get('currentWindow')
     const activeFrame = frameStateUtil.getActiveFrame(currentWindow) || Immutable.Map()
+    const bookmarkDisplayMode = bookmarkUtil.showOnlyFavicon()
+        ? 1
```

Comment:
it's unclear to me what those numbers mean. IMO better to have them under https://github.com/brave/browser-laptop/blob/master/app/common/constants/settingsEnums.js and name them something like `bookmarkDisplayOptions`

Reply:
I don't think that would be appropriate. Those numbers are here purely so that this component (only) will re-render when the mode changes. Putting that in a separate file only spreads logic that is otherwise fully self-contained in to another module. Plus, this is not a settings enum. This value is simply a tri-state value for a prop so that we can compare if the next update has a different display mode. It's irrelevant what the value is, apart from textOnly, which is why that is specified as a prop with a more understandable name.

Reply:
if only a tri-state value, why not use a string value instead? 😄

Reply:
Scratch that... it was appropriate because there is already an enum that I didn't know about, and it's being called many many times for each action / render cycle, so I started using it and then went down a rabbit hole of optimizing it for readability and less state lookups... 🤷‍♀️  ✅